meter/homematic/connection.go

```diff
@@ -130,7 +130,7 @@ func (c *Connection) GridTotalEnergy() (float64, error) {
 	if err != nil {
 		return 0, err
 	}
-	return getFloatValue(res, "IEC_ENERGY_COUNTER") / 1e3, nil
+	return getFloatValue(res, "IEC_ENERGY_COUNTER"), nil
```

Comment:
Den verstehe ich nicht- wie geben Energie doch immer in kWh an. Laut Kommentar sind das Wh. Kommentar falsch?

Reply:
jo, Kommentar ist / war falsch.